packages/block-editor/src/components/block-bindings-indicator/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * WordPress dependencies
+ */
+import { ToolbarItem, ToolbarGroup, Icon } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { connection } from '@wordpress/icons';
+import { _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { store as blockEditorStore } from '../../store';
+
+export default function BlockBindingsIndicator( { clientId } ) {
```

Comment:
How about making the name more explicit? Example:
```suggestion
export default function BlockBindingsToolbarIndicator( { clientId } ) {
```